src/it/java/org/owasp/webgoat/IntegrationTest.java

```diff
@@ -15,23 +14,21 @@
 
 public abstract class IntegrationTest {
 
-  private static String webGoatPort =
-      Objects.requireNonNull(System.getProperty("webgoatport", "8080"));
+  private static String webGoatPort = System.getenv().getOrDefault("WEBGOAT_PORT", "8080");
   private static String webGoatContext =
-      Objects.requireNonNull(System.getProperty("webgoatcontext", "/WebGoat/"));
+      System.getenv().getOrDefault("WEBGOAT_CONTEXT", "/WebGoat/");
 
-  @Getter
-  private static String webWolfPort =
-      Objects.requireNonNull(System.getProperty("webwolfport", "9090"));
+  @Getter private static String webWolfPort = System.getenv().getOrDefault("WEBWOLF_PORT", "9090");
 
   private static String webWolfContext =
-      Objects.requireNonNull(System.getProperty("webwolfcontext", "/WebWolf/"));
+      System.getenv().getOrDefault("WEBWOLF_CONTEXT", "/WebWolf/");
 
-  private static boolean useSSL = false;
+  private static boolean useSSL =
+      Boolean.valueOf(System.getenv().getOrDefault("WEBGOAT_SSLENABLED", "false"));
   private static String webgoatUrl =
       (useSSL ? "https:" : "http:") + "//localhost:" + webGoatPort + webGoatContext;
   private static String webWolfUrl =
-      (useSSL ? "https:" : "http:") + "//localhost:" + webWolfPort + webWolfContext;
+      (useSSL ? "http:" : "http:") + "//localhost:" + webWolfPort + webWolfContext;
```

Comment:
If SSL is not used or doesn't make sense in this case, maybe the ternary operator can be removed.

Reply:
That is true. But maybe we want to support this in the future using another variable for WebWolf?